src/database/migrations/20190905031234-create-notifications.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('Notifications', {
+      id: {
```

Comment:
Kindly add a field that contains the notification recipient like ```recipientId```

Reply:
Thank you maxwell. 
Who should be the recipient? I understand that at the present we do not have department manager.

Reply:
The user the notification is meant for. I think it's needed so we can fetch all notifications belonging to a particular user